internal/lsp/completions/refs/refs.go

```diff
@@ -0,0 +1,224 @@
+package refs
```

Comment:
This package adds the ForModule function that backs all the new ref based completions.